tests/compilation_cache_test.py

```diff
@@ -179,6 +181,22 @@ def test_put_executable(self):
           actual = jax.lib.xla_client.execute_with_python_values(deserialized_executable, inputs_to_executable, backend)
           self.assertEqual(expected, actual)
 
+  def test_pmap(self):
+      cc._cache = None
+      with tempfile.TemporaryDirectory() as tmpdir:
+          cc.initialize_cache(tmpdir)
+          f = pmap(lambda x: x - lax.psum(x, 'i'), axis_name='i')
+          shape = (jax.lib.xla_bridge.device_count(), 4)
+          x = np.arange(prod(shape), dtype=np.float32).reshape(shape)
```

Comment:
Similar to the integration test, you can use a simpler input here: 
`x = np.arange(jax.device_count(), dtype=np.int64)`
(and then do same thing except with int32 for the second one)